src/languages/n1ql.formatter.ts

```diff
@@ -516,7 +516,7 @@ export default class N1qlFormatter extends Formatter {
       reservedJoins,
       reservedDependentClauses,
       reservedLogicalOperators: ['AND', 'OR', 'XOR'],
-      reservedKeywords: dedupe([...reservedKeywords, ...reservedFunctions]),
+      reservedKeywords: dedupe(reservedKeywords.concat(reservedFunctions)),
```

Comment:
As I commented in another PR, I see no need to replace spread operator with concat() calls.

Reply:
the concat is faster than spread for large lists, such as the keyword lists

Reply:
As in any case with performance, the only way to tell is do actually perform a measurement and see if there's a real difference.

Reply:
https://www.measurethat.net/Benchmarks/Show/579/1/arrayprototypeconcat-vs-spread-operator

Reply:
sure I'll use spread